code/modules/mob/living/living_healthscan.dm

```diff
@@ -159,6 +159,7 @@ GLOBAL_LIST_INIT(known_implants, subtypesof(/obj/item/implant))
 
 		//snowflake :3
 		data["lung_ruptured"] = human_target_mob.is_lung_ruptured()
+		data["heart_broken"] = human_target_mob.is_heart_broken()
```

Comment:
With how this is currently used, you need to also check `check_tod()` because otherwise an actually unrevivable person that is also heart broken will then say `Myocardial rupture, surgical intervention required` instead of just `Permanently deceased`